src/App.js

```diff
@@ -20,6 +20,16 @@ const parseJwt = ( token ) => {
   return JSON.parse( atob( base64 ) );
 }
 
+const Footer = () => {
```

Comment:
Let's make this into a new file like we have for the components/Header, then import it in App.js